<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAddDataToDatabase">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>152</y>
      <width>232</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton</string>
    </property>
    <property name="text">
     <string>Add data to database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSearchDatabase">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>152</y>
      <width>232</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton</string>
    </property>
    <property name="text">
     <string>Search database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnectToDB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>232</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>ConnectToDB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDisconnectFromDB">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>232</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>DisconnectFromDB</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
